packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.ts

```diff
@@ -15,12 +15,14 @@ import {
   defaultAuthHubHandler,
   listenToAuthHub,
   SocialProvider,
-  translate,
+  authenticatorTextUtil,
 } from '@aws-amplify/ui';
 import { AmplifySlotDirective } from '../../../../utilities/amplify-slot/amplify-slot.directive';
 import { CustomComponentsService } from '../../../../services/custom-components.service';
 import { AuthenticatorService } from '../../../../services/authenticator.service';
 
+const { getSignInTabText, getSignUpTabText } = authenticatorTextUtil;
```

Comment:
`authenticatorTextUtil` (see code below) centralizes all Authenticator texts we have. Calling one of these utils will get that string, and translate it.